testing/features/step_definitions/buttons_steps.rb

```diff
@@ -28,5 +28,10 @@
 end
 
 Then("the Tertiary button text is less prominent than the {button-type} Button") do |button|
-  expect(@component.font_weight_of("tertiary")).to be < @component.font_weight_of(button)
+  if safari?
+    expect(@component.font_weight_of("tertiary")).to eq("normal")
+    expect(@component.font_weight_of(button)).to eq("bold")
```

Comment:
why do we test appearance in the behavioural tests? This makes them brittle. Would using Backstop for this make more sense?

Reply:
💯 agree

Maybe we can just delete it from this suite of tests? It will already be covered by the VR tests, right?

Reply:
Yes, behaviour like that will be much better tested with the VRTs as they check what the font actually looks like vs whether the CSS attemptes to make it bold or not.

Reply:
I commented this non-behaviour scenario about the font-weight, just in case Luke wanted to use it for another purpose. If not, we are going to remove it permanently. In this PR there is another fix in a behavioural scenario and some changes in local.rb, I'm keeping these changes.

Reply:
I'm happy to remove this scenario. Think we need a follow-up PR to tidy up the safari bit below.